### ISSUE TITLE
Volumes/Secrets not mounting correctly on Windows

### ISSUE THREAD

**Issue by mgj** [telepresenceio/telepresence#2564]
**Describe the bug**
I am running a kubernetes cluster with minikube on windows, and can successfully create an intercept using Telepresence. I can also successfully get Telepresence to export environment variables to an environments file. With VSCode I can use the Telepresence intercept to hit debug breakpoints in my code.

However, even though the intercept appears to be created correctly, no volumes/secrets are actually mounted locally. Specifying a custom mount point using `--mount=C:\mymount` also appear to succeed but again no volumes/secrets are actually mounted - the directory is empty.

The custom mount point directory appears to be deleted after quitting Telepresence, if I manually create it before starting the intercept. If I don't create it manually, it is not created by Telepresence.

[telepresence-logs.zip](https://github.com/telepresenceio/telepresence/files/8645208/telepresence-logs.zip)

**To Reproduce**
1. When I run `telepresence intercept myownapp-myk8sservice-web --port 9090:http --env-file=go-debug.env`
2. I see 

```
Using Deployment myownapp-myk8sservice-web
intercepted
    Intercept name         : myownapp-myk8sservice-web
    State                  : ACTIVE
    Workload kind          : Deployment
    Destination            : 127.0.0.1:9090
    Service Port Identifier: http
    Volume Mount Point     : T:
    Intercepting           : all TCP connections
```

4. So I look at `T:`
5. See error that `T:` does not exist, and in VSCode I can verify that secrets are not available. Removing the Telepresence intercept and letting the pod run "normally" it is able to find and access the needed secrets.

**Expected behavior**
Expected to be able to see the volume mount point and all secrets mounted in it. Expected to be able to access secrets from my application that is running locally.

**Versions (please complete the following information):**
Windows: Windows 10 Pro Version 10.0.19044 Build 19044
Minikube: v1.25.2
Telepresence Client: v2.5.8 (api v3)
Telepresence Root Daemon: v2.5.8 (api v3)
Telepresence User Daemon: v2.5.8 (api v3)

**VPN-related bugs:**
I am not using VPN

**Additional context**
Minikube is started with `minikube start --driver=docker`. Docker desktop is installed. I am not using WSL.

Possibly relevant github issue: https://github.com/telepresenceio/telepresence/issues/2282
Possibly relevant stackoverflow question: https://stackoverflow.com/questions/71503677/volume-mount-not-working-with-telepresence-intercept

**Comment by cindymullins-dw** [comment]
The volume mount for Windows issue was worked on in July of this year, so should be available on a subsequent release. 2.7 or later, if you could try that.

**Comment by Dakror** [comment]
Using 2.20.0 mounting on windows does not seem to work. the T: drive is not available

**Comment by thallgren** [comment]
Hey @Dakror , please create a new ticket for this problem and provide some more information there such as the logs or other hints that be helpful tracking this down. We can't do much based on what you write here. Mounting works fine in my Windows setup.
